Unfortunately, I'm not able to run `pre-commit` successfully for some reason. Here's what appears to be the relevant chunk from the log:

          _ruamel_yaml.c:6:10: fatal error: Python.h: No such file or directory
              6 | #include "Python.h"
                |          ^~~~~~~~~~
          compilation terminated.
          error: command '/usr/bin/gcc' failed with exit code 1
          [end of output]

      note: This error originates from a subprocess, and is likely not a problem with pip.
    error: legacy-install-failure

    × Encountered error while trying to install package.
    ╰─> ruamel.yaml.clib

--- a/adafruit_74hc595.py
+++ b/adafruit_74hc595.py
@@ -120,7 +120,7 @@ class DigitalInOut:
 
 
 class ShiftRegister74HC595:
-    """Initialise the 74HC595 on specified SPI bus, indicate the 
+    """Initialise the 74HC595 on specified SPI bus, indicate the
     number of shift registers being used and optional baudrate.
     """
 

